### PR TITLE
Included Brazilian Portuguese

### PR DESCRIPTION
**Pull request by nemiro-TECH**
Included Brazilian Portuguese as a fork from Portuguese Language.

Some changes already implemented.

Motivation: If you look in most softwares there are this two options once there are several diferences for this two languages.

### CODE REVIEW

File: sentences/pt_br/_common.yaml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      - in: "azul"
+        out: "blue"
+      - in: "roxo"
+        out: "purple"
+      - in: "roxa"
+        out: "purple"
+      - in: "castanho"
+        out: "brown"
+      - in: "castanha"
+        out: "brown"
```
Comment (guieiras) [inline sentences/pt_br/_common.yaml:42]:
```suggestion
      - in: "castanha"
        out: "brown"
      - in: "marrom"
        out: "brown"
```

File: sentences/pt_br/climate_HassClimateGetTemperature.yaml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+language: pt_br
+intents:
+  HassClimateSetTemperature:
```
Comment (guieiras) [inline sentences/pt_br/climate_HassClimateGetTemperature.yaml:3]: I guess this file was copied from a wrong reference. This intent here should be `HassClimateGetTemperature`

File: sentences/pt_br/climate_HassClimateSetTemperature.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+language: pt_br
+intents:
+  HassClimateSetTemperature:
+    data:
+      - sentences:
+          - "(põe | ponha) <zona> a <temperatura>"
+          - "(põe | ponha) a temperatura a <temperatura>"
+          - "(põe | ponha) a temperatura <na_zona> a <temperatura>"
```
Comment (guieiras) [inline sentences/pt_br/climate_HassClimateSetTemperature.yaml:8]: (personal use) I would like to give more suggestions here:

```suggestion
          - "(mude | muda | mudar | ajustar | ajuste | ajusta | põe | ponha) <zona> (para | a | em) <temperatura>"
          - "(mude | muda | mudar | ajustar | ajuste | ajusta | põe | ponha) a temperatura (para | a | em) <temperatura>"
          - "(mude | muda | mudar | ajustar | ajuste | ajusta | põe | ponha) a temperatura <na_zona> (para | a | em) <temperatura>"
```

**Comment by synesthesiam** [comment]
pt-BR is now in

**Pull request closed without merging**